data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/http/WebClientCloudEventSender.java

```diff
@@ -95,7 +94,7 @@ private void send(final CloudEvent event, final Promise<HttpResponse<Buffer>> br
         if (isRetryableStatusCode(response.statusCode())) {
           logError(event, response);
           breaker.tryFail(new ResponseFailureException(
-            (java.net.http.HttpResponse) response,
```

Comment:
:man_facepalming:  these kind of casts ...

can we introduce a rule to ban that package?